recipes/coz/all/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.1)
+project(coz CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(NOT CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 11)
+endif()
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+find_package(Threads REQUIRED)
+
+file(GLOB_RECURSE sources source_subfolder/libcoz/*.cpp)
+file(GLOB_RECURSE headers source_subfolder/include/*.h)
+
+add_library(${PROJECT_NAME} SHARED ${sources})
+set_target_properties(${PROJECT_NAME} PROPERTIES
+    PUBLIC_HEADER "${headers}"
+    VERSION ${CONAN_PACKAGE_VERSION})
+target_include_directories(${PROJECT_NAME}
+    PUBLIC source_subfolder/libcoz
+    PRIVATE source_subfolder/include)
+target_link_libraries(${PROJECT_NAME} PUBLIC dl rt Threads::Threads elf++ dwarf++)
```

Comment:
Pthread and rt are missing for package info

Reply:
Should self and drawf be atted to the required? They are available in CCI as I recall